scipy/stats/_multivariate.py

```diff
@@ -816,6 +816,18 @@ def __init__(self, mean=None, cov=1, allow_singular=False, seed=None,
             Relative error tolerance for the cumulative distribution function
             (default 1e-5)
 
+    Attributes
```

Comment:
This whole section needs to be indented 4 more spaces to align with its surroundings.